parts/k8s/kuberneteswindowssetup.ps1

```diff
@@ -354,7 +354,7 @@ try
                 -MasterSubnet $global:MasterSubnet `
                 -KubeServiceCIDR $global:KubeServiceCIDR `
                 -VNetCIDR $global:VNetCIDR `
-                -TargetEnvironment $TargetEnvironment
+                -IsAzureStack {{if IsAzureStackCloud}}$true{{else}}$false{{end}}

```

Comment:
@marosset what's the value of creating (and using) a new variable to represent the true/false Azure Stack context? why not just inject the `$true` or `$false` values via go template data model introspection?

Reply:
Mostly i wanted a comment near that explaining what was happening.
I'd be ok just adding a comment here with the if/else left inline.

Reply:
but i guess this is cleaned up a lot since the initial PR so we don't need a comment :)

Reply:
All good I heart comments :)